### ISSUE TITLE
A GNN-based Meta-Learning Method for Sparse Portfolio Optimization

### ISSUE THREAD

**Issue by kayuksel** [adaptive-intelligent-robotics/QDax#135]
Hello,

Let me start by saying that I am a fan of your work here. I have recently open-sourced by GNN-based meta-learning method for optimization. I have applied it to the sparse index-tracking problem from real-world (after an initial benchmarking on Schwefel function), and it seems to outperform Fast CMA-ES significantly both in terms of producing robust solutions on the blind test set and also in terms of time (total duration and iterations) and space complexity. I include [the link to my repository](https://github.com/kayuksel/generative-opt/) here, in case you would consider adding the method or the benchmarking problem to your repository. Note: GNN, which learns how to generate populations of solutions at each iteration, is trained using gradients retrieved from the loss function, as opposed to black-box ones.

Sincerely, K
